Editor/TextProvider.cs

```diff
@@ -192,6 +192,22 @@ internal sealed class TextProvider
             { Strings.LandingPageSampleHeader, "Try our Sample Game" },
             { Strings.LandingPageSampleDescription, "Explore Amazon GameLift with our sample multiplayer game. View integration code, set up hosting with Amazon GameLift Anywhere or Managed EC2 fleets, and experiment with hosting features. Import the sample game into your project, and look for it in the project Assets." },
             { Strings.LandingPageSampleButton, "Import Sample Game" },
+            
+            { Strings.HelpPageTitle, "Help & Documentation" },
+            { Strings.HelpPageDescription, "Learn how to integrate and deploy games for hosting on GameLift. The GameLift service manages game server deployment, operation, and scaling. You can deploy custom-built game servers or use GameLift Realtime Servers to configure lightweight game servers for your game." },
+            { Strings.HelpPageReportIssueLink, "Report Issues \uf086" },
```

Comment:
The mocks have the external link icon (I presume is `\uf086`) always right aligned against the edge of the card whereas this places it right next to the text

Reply:
Updated this